growth/too/views.py

```diff
@@ -783,6 +783,18 @@ def plan_manual():
             telescope = form.telescope.data
             json_data, queue_name = get_json_data_manual(form)
 
+            mess = 'Target list empty. None of the requested fields ' + \
+                   'have references in the band(s) requested.'
+
+            if not json_data["targets"] and bool(form.references.data):
+                flash(mess, 'danger')
+                return render_template('plan_manual.html', form=form,
+                                       telescopes=models.Telescope.query)
+            elif not json_data["targets"]:
+                flash('Target list empty.', 'danger')
```

Comment:
Again, be more clear about what has failed. "No observations were scheduled because..."

Reply:
fixed